jupyter_server_proxy/handlers.py

```diff
@@ -542,6 +578,15 @@ def rewrite_pe(rewritable_response: RewritableResponse):
             self._headers = httputil.HTTPHeaders()
             for header, v in rewritten_response.headers.get_all():
                 if header not in ("Content-Length", "Transfer-Encoding", "Connection"):
+                    # Rewrite Location header in redirects to preserve proxy prefix
```

Comment:
I think this should only be rewritten if `absolute_url == False`:
https://jupyter-server-proxy.readthedocs.io/en/stable/server-process.html#absolute-url